README.md

```diff
@@ -139,6 +139,13 @@ The list of Contributors is automatically generated.
             <sub style="font-size:14px"><b>Joe Landman</b></sub>
         </a>
     </td>
+    <td align="center" style="word-wrap: break-word; width: 150.0; height: 150.0">
+        <a href=https://github.com/eternalfrustation>
+            <img src=https://avatars.githubusercontent.com/u/64073903?v=4 width="100;"  style="border-radius:50%;align-items:center;justify-content:center;overflow:hidden;padding-top:10px" alt=Eternal Frustation/>
+            <br />
+            <sub style="font-size:14px"><b>Eternal Frustation</b></sub>
+        </a>
+    </td>
```

Comment:
You don't have to change that. It's automatically generated by the CI.

https://github.com/niklas-heer/speed-comparison/blob/master/.github/workflows/contributors.yml